src/components/Onboarding.tsx

```diff
@@ -121,7 +126,15 @@ export class Onboarding extends React.PureComponent<OnboardingProps, OnboardingS
 
   onScrollEnd(e: NativeSyntheticEvent<NativeScrollEvent>) {
     const offset = e.nativeEvent.contentOffset.x;
-    this.setState({ currentPageIndex: Math.floor(offset / this.state.pageWidth) });
+    const newPageIndex = Math.floor(offset / this.state.pageWidth);
+    this.setState({ currentPageIndex: newPageIndex });
+    this.props.updateParentIndex?.(newPageIndex);
+  }
+
+  scrollPage(direction: ScrollDirection, index: number) {
```

Comment:
since this is an imperative command, I'd rename it to `scrollToPage` (like `scrollTo` or `scrollToEnd` in the underlying `ScrollView`)

Reply:
Commit 9190d6b Thanks a lot